### DISCUSSION TITLE
What do newcomers need to know about design systems in order to succeed in this team?

### DISCUSSION THREAD

**Discussion by kelenelee** [hackforla/design-systems#414] [Polls]
Jennie Yip's conference talk about Atlassian's design system has been a required watch for new people to the DS team since about January, when the DS team got a lot of newcomers, and started focusing on the user interview study. Let's reflect on whether we should continue to require this video for newcomers.

Update: due to the results of this poll, which 5 people took during the 4/25 all-hands meeting, we have removed the Atlassian video from newbie required readings. 

**Now the question becomes:** _what do DS team members need to know in order to succeed in their work?_ Is it knowledge about what design systems are? How they are implemented in the context of a company? Or do we need to prep people more on the team's project history? 

- #412

**Comment by kelenelee** [comment]
### If you said "I would be equally successful learning the same concepts elsewhere"

where would that elsewhere be? or, what do you wish you were taught?

**Reply by sandraberjan** [reply]
There are a few points that Yip makes which could easily be made into a deck and that I do find make me successful. I think that her talk as a whole can be overwhelming to someone just starting out to understand design systems. The points I took away are: design systems are malleable and sometimes hard to look at as a whole, but there are always certain practices in play, even if we can't pinpoint them. A design system team needs to first identify how things are done, what decisions are assumed, what is simply implied and THEN try to standardise these somewhat implicit processes and replace them by an explicit structure that anyone can look up and be guided by.

**Reply by kelenelee** [reply]
Your takeaways from the video match what I and other teammates learned from it. If the video's relevant content can be summarized in a paragraph, then requiring every newcomer to watch it ... is probably not an efficient use of time

**Comment by kelenelee** [comment]
### If you said "I would be less successful on this team"

what is unique about that video's teachings?

**Comment by sandraberjan** [comment]
I appreciate that we are examining if this talk is actually necessary during onboarding. It is fantastic and I think it is useful at some point, but I am not sure that onboarding is the time to view it.
